electrum/gui/qt/send_tab.py

```diff
@@ -297,226 +308,158 @@ def get_frozen_balance_str(self) -> Optional[str]:
         return self.format_amount_and_units(frozen_bal)
 
     def do_clear(self):
-        self._lnurl_data = None
+        self.lock_fields(lock_recipient=False, lock_amount=False, lock_max=True, lock_description=False)
         self.max_button.setChecked(False)
-        self.payment_request = None
-        self.payto_URI = None
         self.payto_e.do_clear()
-        self.set_onchain(False)
-        for e in [self.message_e, self.amount_e]:
+        for w in [self.comment_e, self.comment_label]:
+            w.setVisible(False)
+        for e in [self.message_e, self.amount_e, self.fiat_send_e]:
             e.setText('')
-            e.setFrozen(False)
-        for e in [self.send_button, self.save_button, self.clear_button, self.amount_e, self.fiat_send_e]:
-            e.setEnabled(True)
+            self.set_field_style(e, None, False)
+        for e in [self.save_button, self.send_button]:
+            e.setEnabled(False)
         self.window.update_status()
         run_hook('do_clear', self)
 
-    def set_onchain(self, b):
-        self._is_onchain = b
-        self.max_button.setEnabled(b)
-
-    def lock_amount(self, b: bool) -> None:
-        self.amount_e.setFrozen(b)
-        self.max_button.setEnabled(not b)
-
     def prepare_for_send_tab_network_lookup(self):
-        self.window.show_send_tab()
-        self.payto_e.disable_checks = True
-        for e in [self.payto_e, self.message_e]:
-            e.setFrozen(True)
-        self.lock_amount(True)
+        self.window.show_send_tab() # FIXME why is this here
+        #for e in [self.payto_e, self.message_e]:
+        # self.payto_e.setFrozen(True)
         for btn in [self.save_button, self.send_button, self.clear_button]:
             btn.setEnabled(False)
```

Comment:
try pasting an *mainnet* lnurl-pay into the qt payto_e, when running with `--testnet`.
When you click `Pay...`, you get error: `Wrong Lightning invoice HRP bc1u, should be tb`, after which all buttons are disabled (including `Clear`) and the fields are stuck -> forcing the user to restart the program.